<launch>
    <arg name="data_path" value="/home/pharos/test2_ws/data/test2/"/>

	<node type="rviz" name="rviz" pkg="rviz" args="-d $(find slam)/config/heuristic_slam.rviz" output="log" />

	<node name="slam" pkg="slam" type="heuristic_slam_node" output="screen">

		<rosparam command="load" file="$(arg data_path)param.yaml" />
		<param name="data_path" value="$(arg data_path)" />		

	</node>
</launch>